### ISSUE TITLE
PUT /org/{orgId}/users/{userId} requires "_type": "user" to work

### ISSUE THREAD

**Issue by ijxy** [vleipnik-consensys/tmnt-client#20]
When calling the above from the Swagger UI or via the nodejs client, it only works if the update contains a seemingly random field `"_type": "user"`.

Raising this as it seems likely to be a bug.

**Comment by vleipnik** [comment]
@parkinsonj I agree this is a bug, albeit not a major one. We will schedule it to be fixed.

**Comment by ijxy** [comment]
Awesome, thanks a lot for fixing these issues so quickly. 👍
